conseil-common/src/test/scala/tech/cryptonomic/conseil/common/bitcoin/rpc/BitcoinRpcClientTest.scala

```diff
@@ -61,10 +62,10 @@ class BitcoinRpcClientTest extends WordSpec with Matchers with BitcoinFixtures {
 
   trait BitcoinClientFixtures {
 
-    def bitcoinClient(json: String): BitcoinClient[IO] = {
+    def bitcoinClient(jsonResponse: String): BitcoinClient[IO] = {
```

Comment:
Would calling this something like `bitcoinClientStub` or `bitcoinClientFake`... make the role of this method clearer?